mirascope/core/base/_call_factory.py

```diff
@@ -48,7 +48,7 @@
 _ResponseChunkT = TypeVar("_ResponseChunkT")
 
 
-def call_factory(
+def call_factory(  # noqa: ANN202
```

Comment:
is it worth adding a protocol here instead of ignoring?

Reply:
I tried but could not resolve the type of setupcall properly.
We will address this after we resolve the setupcall.